### ISSUE TITLE
1.0.2

### ISSUE THREAD

**Issue by japlab** [jnxxx/homeassistant-dabblerdk_powermeterreader#6]
Hi Jan
I updated to the new version and get error.
Can you please help with some advise for me.
BR Jan O Petersen

![Skærmbillede 2023-01-18 kl  23 40 04](https://user-images.githubusercontent.com/9386193/213311159-5dbf81de-4fe0-45d4-bd72-2289e772bccc.png)

**Comment by jnxxx** [comment]
Hi Jan,

What version of Home Assistant core are you running?

The HA team is changing some constants. This is one of them.
Currently both the old and new ones exist, but old one with a comment:
```
ENERGY_KILO_WATT_HOUR: Final = "kWh"
"""Deprecated: please use UnitOfEnergy.KILO_WATT_HOUR."""
```
See: [https://github.com/home-assistant/core/blob/9631146745163dbf08ae22f34d0e5d2164a22b4e/homeassistant/const.py#L515](https://github.com/home-assistant/core/blob/9631146745163dbf08ae22f34d0e5d2164a22b4e/homeassistant/const.py#L515)
So, I switched to the new one.

It seems it appeared in October. Probably released in 2022.11.1.

**Comment by japlab** [comment]
I was running core 2022.10.5 and updated to 2023.1.5 and problem solved.
I appreciate you kindly help and thank you very much.
Hope to see the annoying spikes disappear 
Best regards Jan P
